packages/protocol/contracts/thirdparty/risczero/groth16/Groth16Verifier.sol

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
Same as above. Not needed to bring it over as a copy, reference with `npm` or `git` `submodule`.

Reply:
Used pnpm package to keep it outside.